### PR TITLE
Add ADR for Standards

### PR DESCRIPTION
**Pull request by jamietanna**
As we are defining the schema for the Standards portion of the
Workbench, we should make it clear that we've got an Architecture Design
Review (ADR) to capture this.

As we have discussed it offline, we are going straight to mark it as
`accepted`.

### CODE REVIEW

File: adr/0001-standards-catalogue-data-structure.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+decision_date: null
+status: proposed
+---
+# Define the schema for the standards catalogue
+
+## Context
+
+The Standards Catalogue, which sits under the Data Standards Authority Workbench (DSA Workbench), is used to capture standards that the DSA are looking at, before they become official guidance on Gov.UK.
+
+As part of plans to enhance the Workbench by produce schema.org metadata, we discovered that the underlying data wasn't quite standardised.
```
Comment (charlesbaird-dsa) [inline adr/0001-standards-catalogue-data-structure.md:12]:
```suggestion
As part of plans to enhance the Workbench by introducing schema.org metadata, we discovered that the underlying data wasn't quite standardised.
```

**Review by charlesbaird-dsa — COMMENTED** [review]
As discussed.

**Review by charlesbaird-dsa — APPROVED** [review]
Ah nice, you caught the mention of 'catalogue' I missed. Intentional, obvs...

File: adr/0001-standards-catalogue-data-structure.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+creation_date: 2021-11-10
+decision_date: 2021-11-10
+status: accepted
+---
+# Define the schema for the standards portion of the Data Standards Authority Workbench
+
+## Context
+
+The standards section of the Data Standards Authority Workbench (DSA Workbench), is used to capture standards that the DSA are looking at, before they become official guidance on Gov.UK. It is also an attempt to map other standards in use across government, both for the DSA's internal prioritisation activities and to enable external users to find relevant data standards. To begin with, this second focus is served by curating a list of standards catalogues.
```
Comment (charlesbaird-dsa) [inline adr/0001-standards-catalogue-data-structure.md:10]:
```suggestion
The standards section of the Data Standards Authority Workbench (DSA Workbench) is used to capture standards that the DSA are reviewing, before they become official guidance on Gov.UK. It is also an attempt to map other standards in use across government, both for the DSA's internal prioritisation activities and to enable external users to find relevant data standards. To begin with, this second focus is served by curating a list of standards catalogues.
```

**Review by charlesbaird-dsa — APPROVED** [review]
Sorry, I got bothered by a comma. Other than that lgtm.

File: adr/0001-standards-catalogue-data-structure.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+*identifier
+name {label: "string, required"}
+url {label: "url, required"}
+
+Standard 1--* Standard
+Standard 1--+ Topic
+Standard 1--1 Licence
+Standard 1--1 Organisation
+Standard 1--1 Status
+```
```
Comment (arnau) [inline adr/0001-standards-catalogue-data-structure.md:75]: Tweaked the ERD to express the relationship betwen Standard - Event - Status:

```suggestion
[Standard]
*identifier
name { label: "string, required" }
contraction { label: "string, required" }
specification_url { label: "url, required" }
description { label: "markdown" }
+topic_id
+licence_id
+maintainer_id

[Event]
+standard_id
+status_id
date {label: "required, date"}

[Status]
*identifier
name
description

[Tag]
*identifier
name
description

[Licence]
*identifier {label: "string, required"}
name {label: "string, required"}
type {label: "enum[OPEN,PROPRIETARY,UNKNOWN]"}
url {label: "url"}

[Organisation]
*identifier
name {label: "string, required"}
url {label: "url, required"}

Standard 1--* Standard
Standard 1--+ Tag
Standard 1--1 Licence
Standard 1--1 Organisation
Standard 1--+ Event
Event 1--1 Status
```

File: adr/0001-standards-catalogue-data-structure.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+</details>
+
+## Consequences
+
+TBC
```
Comment (jamietanna) [inline adr/0001-standards-catalogue-data-structure.md:81]: needs addressing

**Review by arnau — CHANGES_REQUESTED** [review]

**Review by arnau — APPROVED** [review]
Nice one! 🚢

**Pull request merged**